Units/parser-javascript.r/js-es6-class.d/expected.tags

```diff
@@ -4,11 +4,17 @@ Class2	input.js	/^class Class2$/;"	c
 Class3	input.js	/^var Class3 = class {$/;"	c
 Class4	input.js	/^var Class4 = class Class4_2 {$/;"	c
 Class4_2	input.js	/^var Class4 = class Class4_2 {$/;"	c
+InnerClass1	input.js	/^	class InnerClass1 {$/;"	c	class:func1
+InnerClass2	input.js	/^	class InnerClass2 {$/;"	c	class:func1
+func1	input.js	/^function func1() {$/;"	f
+func2	input.js	/^function func2() {$/;"	f
 method1	input.js	/^	method1(arg1,arg2)$/;"	m	class:Class1
+method10	input.js	/^		method10() {$/;"	m	class:InnerClass2
 method2	input.js	/^	method2(arg1,arg2)$/;"	m	class:Class1
 method3	input.js	/^	static method3(arg = 10)$/;"	m	class:Class2
 method4	input.js	/^	method4(){}$/;"	m	class:Class3
 method5	input.js	/^	static method5(){}$/;"	m	class:Class3
 method6	input.js	/^	method6(){}$/;"	m	class:Class4
 method7	input.js	/^	static method7(){}$/;"	m	class:Class4
 method8	input.js	/^	method8(n) { return n * n; }$/;"	m	class:AnonymousClass152c525b0101
+method9	input.js	/^		method9() {$/;"	m	class:InnerClass1
```

Comment:
> Specifying just one upper scope object is enough and is the job of ctags.

No, it's not: full scope is useful for knowing the real hierarchy.  Imagine `A.Foo.b` and `B.Foo.b`, how do you discriminate both if not for their full scope?

What you see here is a bug of the JS parser, but AFAIK it's unrelated to the new changes and I was unable to fix it easily yet.